### PR TITLE
[Preview 2 Release]Show warning for registry admin password

### PR DESCRIPTION
**Pull request by loganhz**
Proposed changes
======
Now if users re-use the PVC, the admin password will not take effect. 

Types of changes
======
- Bugfix (non-breaking change which fixes an issue)

Linked Issues
======
https://github.com/rancher/rancher/issues/20797

 

Further comments
======

### CODE REVIEW

File: translations/en-us.yaml [hunk 1]
```diff
@@ -1209,6 +1209,7 @@ globalRegistryPage:
     adminPassword:
       label: Admin Password
       placeholder: Harbor Admin password
+      helpText: "If you re-use an existing persistent volume claim for Database component, the admin password will not take effect."
```
Comment (gitlawr) [inline translations/en-us.yaml]: The password will take effect, if the existing PVC bounds to a fresh PV.
See case 2 in the comment: https://github.com/rancher/rancher/issues/20797#issuecomment-501515382

Comment (loganhz) [reply translations/en-us.yaml]: How about `If you are using the existing PVC which bounds to a used PV, the password will not take effect`

Comment (gitlawr) [reply translations/en-us.yaml]: I prefer `If you are using the existing PVC with initialized Harbor data, the password will not take effect`

**Comment by loganhz** [comment]
Updated

**Comment by gitlawr** [comment]
LGTM

**Pull request merged**